src/components/ScatterSeries.tsx

```diff
@@ -64,14 +58,15 @@ function ScatterSeriesRender({
     const Marker = markersComps[markerShape];
     const markers = data.map((point, i) => {
       return (
-        <Marker
-          // eslint-disable-next-line react/no-array-index-key
+        <g // eslint-disable-next-line react/no-array-index-key
           key={`markers-${i}`}
-          x={xScale(point.x)}
-          y={yScale(point.y)}
-          size={markerSize}
-          style={functionalStyle(defaultColor, markerStyle, point, i, data)}
-        />
+          transform={`translate(${xScale(point.x)}, ${yScale(point.y)})`}
+        >
+          <Marker
```

Comment:
there's a bug somewhere. on the scatterplot story, `xmark` and `cross` don't work: https://deploy-preview-125--amazing-kilby-d9c1d7.netlify.app/?path=/story/api-scatter--scatter-control